libraries/helpers.rb

```diff
@@ -20,32 +20,42 @@
 module CDAP
   module Helpers
     #
+    # Return true if Explore is enabled
+    #
+    def cdap_explore?
+      cdap_property('explore.enabled')
```

Comment:
Then `cdap_explore?` doesn't return true, and `explore.enabled` gets written in the template as false.